streaming/src/main/java/org/openmrs/analytics/FhirStreaming.java

```diff
@@ -26,14 +27,33 @@
  */
 public class FhirStreaming {
 	
-	private static final Logger log = LoggerFactory.getLogger(FhirStreaming.class);
+	private static final Logger log = LoggerFactory.getLogger(FhirEventWorker.class);
+	
+	// TODO: set as arg or env variable? using constant for simplicity
+	private static final String FEED_ENDPOINT = "/ws/atomfeed";
+	
+	private static final String FHIR_ENDPOINT = "/ws/atomfeed";
```

Comment:
Is this correct? Shouldn't this be `/ws/fhir2` instead?

Reply:
I'll get rid of the fix - it should be in the config.